server/src/main/java/org/opentosca/toscana/core/parse/converter/NodeConverter.java

```diff
@@ -0,0 +1,149 @@
+package org.opentosca.toscana.core.parse.converter;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Set;
+import java.util.function.BiFunction;
+
+import org.opentosca.toscana.model.node.Apache;
+import org.opentosca.toscana.model.node.BlockStorage;
+import org.opentosca.toscana.model.node.Compute;
+import org.opentosca.toscana.model.node.ContainerApplication;
+import org.opentosca.toscana.model.node.ContainerRuntime;
+import org.opentosca.toscana.model.node.Database;
+import org.opentosca.toscana.model.node.Dbms;
+import org.opentosca.toscana.model.node.DockerApplication;
+import org.opentosca.toscana.model.node.LoadBalancer;
+import org.opentosca.toscana.model.node.MysqlDbms;
+import org.opentosca.toscana.model.node.Nodejs;
+import org.opentosca.toscana.model.node.ObjectStorage;
+import org.opentosca.toscana.model.node.RootNode;
+import org.opentosca.toscana.model.node.SoftwareComponent;
+import org.opentosca.toscana.model.node.WebApplication;
+import org.opentosca.toscana.model.node.WebServer;
+import org.opentosca.toscana.model.node.WordPress;
+
+import com.google.common.collect.Sets;
+import org.eclipse.winery.model.tosca.yaml.TNodeTemplate;
+
+public class NodeConverter {
+
+    public static final String TOSCA_PREFIX = "tosca.nodes.";
+
+    private Map<String, BiFunction<String, TNodeTemplate, RootNode>> conversionMap = new HashMap<>();
+
+    NodeConverter() {
+        addRule("Compute", this::toCompute);
```

Comment:
A short comment Explaining the meaning of these parameters would be good

Reply:
Fixed